src/components/Fullscreen.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {FC, useState, useEffect} from 'react';
+import Box from './Box';
+
+const Fullscreen: FC = ({children}) => {
+	const [size, setSize] = useState({
+		columns: process.stdout.columns,
+		rows: process.stdout.rows
+	});
+
+	useEffect(() => {
+		function onResize() {
```

Comment:
Use arrow function.

Reply:
Is this for techincal reasons or just the project's coding standard? (just curious)

Reply:
Just a preference. Arrow functions are better as they don't get hoisted in the scope. Not a problem here, but in general better to be consistent.